### PR TITLE
Fixes for tests against tpm2-tss master

### PR DESCRIPTION
**Pull request by whooo**
Handle recent changes in tpm2-tss master so our tests don't break.

### CODE REVIEW

**Review by williamcroberts — COMMENTED** [review]
@JuergenReppSIT could you once over this for me?

File: test/test_fapi.py [hunk 1]
```diff
@@ -48,6 +50,8 @@ def fapi_config_rsa(simulator):
         tcti=simulator.tcti_name_conf,
         ek_cert_less="yes",
         profile_name="P_RSA2048SHA256",
+        firmware_log_file="/dev/null",
+        ima_log_file="/dev/null",
```
[the remark this answers is not in the document]
Comment (JuergenReppSIT) [reply test/test_fapi.py:54]: With /dev/null for the log files FAPI will produce an empty event list.

[the remark this answers is not in the document]
Comment (whooo) [reply test/test_fapi.py:54]: It isn't for logging, but unless they are set to /dev/null FAPI might parse /sys/kernel/security/ima/binary_runtime_measurements and /sys/kernel/security/tpm0/binary_bios_measurements on the local machine running the tests which could make future tests unstable

[the remark this answers is not in the document]
Comment (williamcroberts) [reply test/test_fapi.py:54]: > It isn't for logging, but unless they are set to /dev/null FAPI might parse /sys/kernel/security/ima/binary_runtime_measurements and /sys/kernel/security/tpm0/binary_bios_measurements on the local machine running the tests which could make future tests unstable

Ahh that makes sense.

**Review by williamcroberts — APPROVED** [review]

**Pull request merged**